prog/main.go

```diff
@@ -119,6 +121,25 @@ type appFlags struct {
 	consulInf       string
 }
 
+func logCensoredArgs() {
+	var prettyPrintedArgs string
+	// we show the args followed by the flags which is likely to change the
```

Comment:
surely it would make more sense to show it the other way round - flags precede args, don't they?